internal/caravan/common.lua

```diff
@@ -13,12 +13,16 @@ local to_pen = dfhack.pen.parse
 SOME_PEN = to_pen{ch=':', fg=COLOR_YELLOW}
 ALL_PEN = to_pen{ch='+', fg=COLOR_LIGHTGREEN}
 
-function make_search_key(str)
-    local out = ''
-    for c in str:gmatch("[%w%s]") do
-        out = out .. c:lower()
+function add_words(words, str)
+    for word in str:gmatch("[%w]+") do
+        table.insert(words, word:lower())
     end
-    return out
+end
+
+function make_search_key(str)
```

Comment:
is this "common" function being used in more than once place with this change, or is that a future enhancement?

Reply:
the make_search_key? it's used by other places that don't need to augment the key